docs/source/guides/network_node/staking_guide.rst

```diff
@@ -19,14 +19,23 @@ they only need to perform stake management transactions. Using a hardware wallet
 for stakers since only temporarily access to private keys is required during stake management while providing a higher standard
 of security than software wallets.
 
-Staking Procedure:
+Mainnet Staking Procedure:
 
-1) Install ``nucypher`` on Staker's machine (see :doc:`/guides/installation_guide`)
-2) Establish ethereum account, provider, and, optionally, signer (see `Staking`_)
-3) Request testnet tokens by joining the `Discord server <https://discord.gg/7rmXa3S>`_ and type ``.getfunded <YOUR_STAKER_ETH_ADDRESS>`` in the #testnet-faucet channel
-4) Initialize a new StakeHolder and Stake (see `Initialize a new stakeholder`_)
-5) Initialize a new stake (see `Initialize a new stake`_)
-6) Bond a Worker to a Staker using the worker's ethereum address (see `Bonding a Worker`_)
+#. Install ``nucypher`` on Staker's machine (see :doc:`/guides/installation_guide`)
+#. Obtain a Stake with tokens through participation in :ref:`Worklock <worklock-guide>`
```

Comment:
I'm not sure what makes the most sense for #2. WorkLock is only temporary so we'll need to either update the docs later, list other possibilities, or just leave it ambiguous "Obtain a Stake with tokens"

Reply:
Agreed.

Reply:
What about something like:

```
Obtain a Stake with tokens (initially via :ref:`Worklock <worklock-guide>` at launch)
```

Reply:
✔️